src/emoji/__tests__/emojiSelectors-test.js

```diff
@@ -71,3 +76,41 @@ describe('getAllRealmEmoji', () => {
     expect(getAllRealmEmoji(deepFreeze(state))).toEqual(expectedResult);
   });
 });
+
+describe('getActiveRealmEmojiTransformed and getAllRealmEmojiTransformed', () => {
```

Comment:
Definitely split each selector in a separate test (and separate `describe` group)